DataFormats/Detectors/TPC/include/DataFormatsTPC/ZeroSuppression.h

```diff
@@ -18,6 +18,8 @@
 #include <cstddef> // for size_t
 #endif
 #include "GPUCommonDef.h"
+#include <vector>
+#include <array>
 
```

Comment:
These includes will not work on the GPU, but the ZeroSuppression.h is included on the GPU.

Reply:
Thanks for the quick reply. I did some cleanup and moved the headers again to a separate file, to be used for the zs container.